Library/Homebrew/config.rb

```diff
@@ -27,7 +27,11 @@ def mkpath
 HOMEBREW_CACHE_FORMULA = HOMEBREW_CACHE+"Formula"
 
 unless defined? HOMEBREW_BREW_FILE
```

Comment:
Do we need to keep this logic? Is there any case where `HOMEBREW_BREW_FILE` will be set by other code?